lib/perl5db.t

```diff
@@ -3012,6 +3012,9 @@ SKIP:
     local $ENV{LANG} = "C";
     local $ENV{LC_MESSAGES} = "C";
     local $ENV{LC_ALL} = "C";
+    my $out = `/usr/bin/man --version`;
+    $out !~ /^This system has been minimized/
+        or skip "No man. This system has been minimized...", 1;
```

Comment:
These 3 lines would be better placed 3 lines higher, *i.e.,* just after the other `skip` statements and before the 3 `local $ENV{}` statements.

I think the last 2 lines would be more quickly understood if they were expressed in the positive, rather than in the negative, *i.e.,*
```
    $out =~ /^This system has been minimized/
        and skip "No man. This system has been minimized...", 1;
```

Reply:
If the locale is non-English moving this above the environment changes may produce non-English messages, breaking the test (though I don't know if whatever produces the message pays the space cost of localizing the message.)

Reply:
Yeah it probably should come after the localizations, but I do agree with @jkeenan that the negative match doesn't help readability here. How about

```
if ($out =~ /^This system has been minimized/) {
    skip "No man. This system has been minimized...", 1;
}
```